openslides/motions/static/js/motions/site.js

```diff
@@ -1374,6 +1375,18 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
                     $scope.showVersion(motion.active_version);
                 });
         };
+        // check if user is allowed to see at least one comment field
+        $scope.isAllowedToSeeCommentField = function () {
+            var isAllowed = false;
+            if ($scope.commentsFields.length > 0) {
+                angular.forEach($scope.commentsFields, function(field) {
```

Comment:
You can use a lodash function the breaks after the first truthy return value. https://lodash.com/docs/4.16.2#find